.github/workflows/test.yml

```
@@ -229,3 +237,28 @@ jobs:
+        with:
+          bun-version: 1.3.14
+      - name: Cache bun modules
+        uses: actions/cache@27d5ce7f107fe9357f9df03efb73ab90386fccae
+        with:
+          path: ~/.bun/install/cache
+          key: ${{ runner.os }}-bun-${{ hashFiles('**/bun.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-bun-
+      - run: bun install --frozen-lockfile
+      - run: bun run ci:verify:distrib
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)